src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Commands.xml

```diff
@@ -3,49 +3,82 @@
      Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. -->
 <Commands>
   <Verb Name="add">
-    <Noun Name="source">
+    <Noun Name="source" Version="3.1.200"
+       Description="adds a new package source to your NuGet configuration files."
+       >
```

Comment:
I don't think I've seen this formatting style before. Normally the `>` is on the end of the last line, similar to how `)` is normally at the end of the last parameter in multi-line expressions or method declarations in c#.  Our `NuGet.targets` file keeps the `>` or `/>` on the same line as the last attribute.

Reply:
fixed in next commit.